nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/DpoConversationResource.java

```diff
@@ -1,27 +1,49 @@
 package no.difi.meldingsutveksling.nextmove;
 
-import com.google.common.collect.Maps;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import no.difi.meldingsutveksling.ServiceIdentifier;
 
 import javax.persistence.DiscriminatorValue;
 import javax.persistence.Entity;
 import java.time.LocalDateTime;
+import java.util.Map;
 
 @Entity
 @DiscriminatorValue("DPO")
+@NoArgsConstructor
 @Data
 public class DpoConversationResource extends ConversationResource {
 
-    private String jpId;
-
-    DpoConversationResource() {}
-
     private DpoConversationResource(String conversationId, String senderId, String receiverId) {
-        super(conversationId, senderId, receiverId, ServiceIdentifier.DPO, LocalDateTime.now(), Maps.newHashMap(), Maps.newHashMap());
+        super(conversationId, senderId, receiverId, ServiceIdentifier.DPO);
+    }
+
+    private DpoConversationResource(String conversationId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)